.editorconfig

```diff
@@ -0,0 +1,6 @@
+root = true
```

Comment:
This file should not be checked in.

Reply:
[EditorConfig](https://editorconfig.org/) is a standard way of defining the basic format conventions used and is not specific to a single IDE. I couldn't find any guidelines within the project on what code style was preferred other than the existing code, which is why I thought to add this.

Happy to remove if you don't want it, but it would be nice to have something like this or some linting rules.

Reply:
Good point. GitHub seems to be using it as well, so I guess keeping this makes sense.